tests/compas/datastructures/test_mesh.py

```diff
@@ -235,16 +264,10 @@ def test_is_regular():
 
 def test_is_manifold():
     pass
-    # TODO: this test needs to be passed!
-    # mesh = Mesh.from_off(compas.get('cube.off'))
-    # assert mesh.is_manifold()
-
-    # mesh.add_face([0, 1, 3])
-    # assert not mesh.is_manifold()
 
 
 def test_is_orientable():
```

Comment:
you could mark these as `skip` instead of keeping empty functions (https://docs.pytest.org/en/latest/skipping.html)

Reply:
i will add an implementation but as part of separate PR. just took out the commented code. they were already empty...